### DISCUSSION TITLE
I don't understand how `tox-gh-actions` picks the correct environment

### DISCUSSION THREAD

**Discussion by stdedos** [ymyzk/tox-gh-actions#189] [Q&A] [answered]
### Question

Hello there,

I have this tox.ini https://github.com/pylint-dev/pylint-pytest/blob/300774b/tox.ini and this action https://github.com/pylint-dev/pylint-pytest/blob/300774bc6e7e1612f9753ed97d781f4c42e3d680/.github/workflows/run-tests.yaml#L44.

I am wondering why every other environment is picked up automatically, but the `py36` cannot be picked up.

The `tox -vv` logging between 3.6 and e.g. 3.7 is "much different" from me, and I don't understand what would be blocking understanding that Python3.6 === py36 environment.

**Comment by ymyzk** [comment, marked as answer]
Hello @stdedo . tox-gh-actions always require users to configure `[gh-actions]` in `tox.ini`. For example, if you add something like the following, tox and tox-gh-actions may work as expected.

```ini
[gh-actions]
python =
    3.7: py37
    3.8: py38
    3.9: py39
    3.10: py310
    3.11: py311
    3.12: py312
```

Reference: https://github.com/ymyzk/tox-gh-actions#tox-gh-actions-configuration

**Reply by stdedos** [reply]
I see - I guess I didn't even "really activate it" 😅 
But _somehow_ it looked like it affected results.

In any case - thank you for your response!
